engine/gamesys/src/gamesys/wscript

```diff
@@ -23,7 +23,7 @@ def remove_from_node_list(lst, name):
 def build(bld):
     build_util = create_build_utility(bld.env)
 
-    sources = bld.path.ant_glob(['*.cpp', 'resources/**/*.cpp', 'components/**/*.cpp', 'scripts/*.cpp']) + bld.path.parent.parent.ant_glob('proto/gamesys/*')
+    sources = bld.path.ant_glob(['*.cpp', 'resources/**/*.cpp', 'components/**/*.cpp', 'scripts/*.cpp', 'fontgen/*.cpp']) + bld.path.parent.parent.ant_glob('proto/gamesys/*')
```

Comment:
Q: If I understand correctly we include `fontgen`source every time and doesn't create separate library to include/exclude via appmanifest. Is it correct understanding? If yes - should it be like this?

Reply:
Yes, the idea is that we want to move towards an easier-to-use font system.
At least that is the current goal.
As shown in the numbers table in the OP, using runtime fonts is a net gain, with this file included in those numbers.